research-software-directory/RSD-as-a-service
# Community documentation

## Community documentation

Changes proposed in this pull request:

* Add documentation on maintaining a community, joining a community and community categories

How to test:

* `docker compose down --volumes && docker compose build --parallel && docker compose up --scale data-generation=1`
* Check the following sections:
	* http://localhost/documentation/users/community/
	* http://localhost/documentation/users/adding-software/#categories
	* http://localhost/documentation/users/adding-software/#communities

Closes #1186

PR Checklist:
